### PR TITLE
⬆️ (python) use python 3 to run arnold's playbooks

### LINKED ISSUE #78: Consider switching base image to ubuntu
## Purpose

As mentioned in #76, Arnold's docker image is a tool, so it can be ubuntu-based instead of debian-based. Switching to Ubuntu would allow to use more recent packages, particularly, python 3.6.

## Proposal

- [ ] switch to tiny `ubuntu:18.04` base image
- [ ] update apt package names and versions
- [ ] upgrade python packages versions (?)

### PR DESCRIPTION
**Pull request by lunika**
## Purpose

In #78 it is suggested to switch the base image used to Ubuntu. In this PR I choose python:3.6 as this image will be dedicated to tools using python.

Fixes #78

## Proposal

Upgrade to python 3.6 and adapt tools to be compatible with this version of python
For now it is not possible to upgrade to ansible 2.6 or 2.7 because the `openshift_raw` modules has been deprecated and aliased to `k8s` which is not fully compatible with `openshift_raw` (this is a bit hard for a deprecation...)

Plugins are updated to work with python 3.6, minor changes have been made.

### CODE REVIEW

File: inventory
Comment (lunika) [inline inventory]: I will remove it from here. This configuration is specific to our image so it should be set on calling ansible-playbook inside container, not here.

File: ansible.cfg
Comment (jmaupetit) [inline ansible.cfg:9]: Can you remove this callback plugin in a separated commit?

File: bin/ansible-playbook
Comment (jmaupetit) [inline bin/ansible-playbook]: Can we use an environment variable instead? I know that this configuration is not bundled with Ansible, and that we will have to handle it manually.

**Review by jmaupetit — COMMENTED** [review]
We need to publish a docker image release using python 2.7 before merging this. Even if we do our best to test Arnold, this would be safer.

File: bin/ansible-playbook
Comment (jmaupetit) [reply bin/ansible-playbook]: You can pick ideas here: https://github.com/ansible/ansible/issues/6345

File: ansible.cfg
Comment (lunika) [reply ansible.cfg:9]: :+1:

File: bin/ansible-playbook
Comment (lunika) [reply bin/ansible-playbook]: `ansible_python_interpreter="/usr/bin/env python3"` in the inventory is the solution I choosed

**Comment by jmaupetit** [comment]
This work is blocked until #56 has been implemented.

**Comment by jmaupetit** [comment]
This PR is a major change that requires more testing. To avoid breaking our 1.0.0 release cycle, I think we need to:

1. Fix failing tests in the CI
2. Merge this PR in a `py3` branch
3. Heavily test the new python 3 implementation
4. Merge the `py3` branch in `master` once we are on the 2.0.0 roadmap.

WDYT?

**Comment by jmaupetit** [comment]
After thought: this PR might not be a BC, hence this does not justify postponing it to a 2.0.0 roadmap. I'll fix and test it.

File: CHANGELOG.md
Comment (jmaupetit) [inline CHANGELOG.md]:
```suggestion
- Migrate docker image to python 3.6
```

File: Dockerfile
Comment (jmaupetit) [inline Dockerfile:24]: `curl` is already installed in the `python:3.6` image, right?

**Review by jmaupetit — APPROVED** [review]
LGTM (after a little suggestion for the CHANGELOG).

File: Dockerfile
Comment (lunika) [reply Dockerfile:24]: Yep, this is why I removed it.

**Pull request merged**
